ui/src/app/about/about.service.spec.ts

```diff
@@ -1,21 +1,118 @@
 import { AboutService } from './about.service';
 import { Observable } from 'rxjs/Rx'
 import { ErrorHandler } from '../shared/model/error-handler';
+import { Response, ResponseOptions } from '@angular/http';
 
-xdescribe('AboutService', () => {
+describe('AboutService', () => {
 
   let aboutService: AboutService;
   let mockHttp;
```

Comment:
Should mockHttp be marked with `any` type since the initializing function returns that type?